js/src/dashboard/all-programs-table-card/index.js

```diff
@@ -130,18 +142,23 @@ const AllProgramsTableCard = ( props ) => {
 					{ display: el.country },
 					{ display: el.dailyBudget },
 					{
-						display:
-							el.id === FREE_LISTINGS_PROGRAM_ID ? (
-								<FreeListingsDisabledToggle />
-							) : (
-								<ProgramToggle key={ el.id } program={ el } />
-							),
+						display: isPaidCampaign && (
+							// TODO: Revisit this temporary demo since the spec of assets column is not yet decided.
```

Comment:
❓  Could you elaborate what do you mean with this?

Reply:
The spec of assets column was not yet decided. This column might not be added, or the logic of the column content might still be adjusted. To facilitate the review of this PR, I first made a column for demonstrating entering the asset edit step directly. This `TODO` is used to remind me to come back to handle it, and the reason.